packages/mux-player/src/html.ts

```diff
@@ -46,8 +46,20 @@ export function processBooleanAttribute(
   return false;
 }
 
+export function processBooleanNode(
```

Comment:
Assuming this was added for the "false" cases in rendering with the template parts refactor and is unrelated to this PR?